driver/ppm.h

```diff
@@ -117,4 +117,10 @@ extern const enum ppm_syscall_code g_syscall_ia32_code_routing_table[];
 
 extern void ppm_syscall_get_arguments(struct task_struct *task, struct pt_regs *regs, unsigned long *args);
 
+extern nanoseconds ppm_nsecs(void);
```

Comment:
nit: Make this a `static inline` function instead?

Reply:
I agree, this have to be inline

Reply:
inline won't mean much here. it's a hint to the compiler, not a requirement...and given that it's not defined in this compilation unit, afaik, it's almost assuredly NOT going to end up inlined regardless. how could it? the compiler doesn't have the definition of the function at compile time, and thus would have no idea how much space in the ASM for the function to leave.